Week 2/Code/getUsers.php

```diff
@@ -4,7 +4,7 @@
         'method'=>"GET",
         'header'=>"Accept-language: en\r\n" .
             "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36",
-            "content-type: application/json",
```

Comment:
the tab makes it readable could you add that back in.

Reply:
Hey Brian, I made the fix now on the indentation and push it up. Can you let me know what you think?

Thanks,
Miguel Ravelo